README.md

```diff
@@ -184,14 +184,14 @@ In Device Simulator Express, you can use keyboard to interact with the device:
 
 #### I. Take a look at the "Device Simulator Express: Getting Started" Command.
 1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
-<img alt="Deploy to Device" src='assets/readmeFiles/getting_started.png'>
-2. Choose the `micro:bit` option from the dropdown.
+<img alt='Getting Started' src='https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/getting_started.png'>
+1. Choose the `micro:bit` option from the dropdown.
 2. Read, copy and learn some of the things you can do with the simulator!

```

Comment:
I think this is supposed to be:
1 Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette). 
2 (**instead of 1**) Choose the `micro:bit` option from the dropdown.
3 (**instead of 2**) Read, copy and learn some of the things you can do with the simulator!

Reply:
good catch. updated it :D